consensus/types/src/contribution_and_proof.rs

```diff
@@ -10,7 +10,6 @@ use test_random_derive::TestRandom;
 use tree_hash_derive::TreeHash;
 
 /// A Validators aggregate sync committee contribution and selection proof.
-
```

Comment:
This extra line kept triggering Clippy. Not sure why this wasn't picked up before though